content/ja/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,120 @@
+---
+tille: ワークロードの自動スケーリング
+description: >-
+  自動スケーリングによって、何らかのかたちでワークロードを自動的に更新できます。これによりクラスターはリソース要求の変化に対してより弾力的かつ効率的に対応できるようになります。
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Kubernetesでは、現在のリソース要求に応じてワークロードをスケールできます。
+これによりクラスターはリソース要求の変化に対してより弾力的かつ効率的に対応できるようになります。
+
+ワークロードをスケールするとき、ワークロードによって管理されるレプリカ数を増減したり、レプリカで使用可能なリソースをインプレースで調整できます。
+
+ひとつ目のアプローチは _水平スケーリング_ と呼ばれ、一方でふたつ目のアプローチは _垂直スケーリング_ と呼ばれます。
+
+ユースケースに応じて、ワークロードをスケールするには手動と自動の方法があります。
+
+<!-- body -->
+
+## ワークロードを手動でスケーリングする
+
+Kubernetesはワークロードの _手動スケーリング_ をサポートします。
+水平スケーリングは `kubectl` CLIを使用して行うことができます。
+垂直スケーリングの場合、ワークロードのリソース定義を `パッチ適用` する必要があります。
+
+両方の戦略の例については以下をご覧ください。
+
+- **水平スケーリング**: [Running multiple instances of your app](/docs/tutorials/kubernetes-basics/scale/scale-intro/)
+- **垂直スケーリング**: [Resizing CPU and memory resources assigned to containers](/docs/tasks/configure-pod-container/resize-container-resources)
+
+## ワークロードを自動でスケーリングする
+
+Kubernetesはワークロードの _自動スケーリング_ もサポートしており、これがこのページの焦点です。
+
+Kubernetesにおける _オートスケーリング_ の概念は一連のPodを管理するオブジェクト(例えば{{< glossary_tooltip text="Deployment" term_id="deployment" >}})を自動的に更新する機能を指します。
+
+### ワークロードを水平方向にスケーリングする
+
+Kubernetesにおいて、 _HorizontalPodAutoscaler_ (HPA)を使用してワークロードを水平方向に自動的にスケールできます。
+
+これはKubernetes APIリソースおよび{{< glossary_tooltip text="コントローラー" term_id="controller">}}として実装されておりCPUやメモリ使用率のような観測されたリソース使用率と一致するようにワークロードの{{<glossary_tooltip text="レプリカ" term_id="replica" >}}数を定期的に調整します。
+
+Deployment用のHorizontalPodAutoscalerを構成する[ウォークスルーチュートリアル](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough)があります。
+
+### ワークロードを垂直方向にスケーリングする
+
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+_VerticalPodAutoscaler_ (VPA)を使用してワークロードを垂直方向に自動的にスケールできます。
+HPAと異なり、VPAはデフォルトでKubernetesに付属していませんが、[GitHubで](https://github.com/kubernetes/autoscaler/tree/9f87b78df0f1d6e142234bb32e8acbd71295585a/vertical-pod-autoscaler)見つかる別のプロジェクトです。
+
+インストールすることにより、管理されたレプリカのリソースを _どのように_ _いつ_ スケールするのかを定義するワークロードの{{< glossary_tooltip text="CustomResourceDefinitions" term_id="customresourcedefinition" >}}(CRDs)を作成できるようになります。
+
+{{< note >}}
+HPAが機能するにはクラスターに[Metrics Server](https://github.com/kubernetes-sigs/metrics-server)がインストールされている必要があります。
```

Comment:
BTW, the word of "HPA" is weird at here, because this section said the content about "VPA" not "HPA". :thinking:

Reply:
I know the original `en` content also says "HPA" here.

Reply:
It's just the comment, not request to change here.